packages/firestore/src/api/database.ts

```diff
@@ -371,17 +372,17 @@ export function enableIndexedDbPersistence(
  * shared execution of queries and latency-compensated local document updates
  * across all connected instances.
  *
- * If this fails, `enableMultiTabIndexedDbPersistence()` will reject the promise
- * it returns. Note that even after this failure, the {@link Firestore} instance will
- * remain usable, however offline persistence will be disabled.
- *
- * There are several reasons why this can fail, which can be identified by
- * the `code` on the error.
+ * If fails, `enableMultiTabIndexedDbPersistence()` will reject the promise or
```

Comment:
Please make the same changes to the docs for `enableMultiTabIndexedDbPersistence()` as are made to `enableIndexedDbPersistence()`.